src/services/graph/hooks/exchange.ts

```diff
@@ -188,6 +169,13 @@ export function useMaticPrice(variables = undefined, swrConfig: SWRConfiguration
 
 export function useSushiPrice(variables = undefined, swrConfig: SWRConfiguration = undefined) {
```

Comment:
Do we still need useSushiPrice ?

Reply:
Now `useSushiPrice` is only referenced by `useFarmReward`, which is used in `/analytics` page. 

I can try to replace the use of `useSushiXXX` with `useMistXXX`  in `useFarmReward`.

Reply:
I think we would better not remove `useSushiPrice`  now.  There are other code named `SuShi` in `/analytics`. The code could not be clean without removing `analytics` related pages.